tests/unit/services/best-language-test.ts

```diff
@@ -67,6 +67,23 @@ describe('Unit | Service | best-language', () => {
           });
         });
 
+        it('should return the closest match when provided with languages with country codes', function() {
+          const service: BestLanguage = this.owner.lookup(
+            'service:best-language'
+          );
+
+          const supportedLanguages = ['en-CA', 'fr'];
+
+          const result = service.bestLanguage(supportedLanguages);
+          console.log(JSON.stringify(result));
```

Comment:
`console.log` should be breaking the CI build 🤔

Reply:
Build isn’t finished so I don’t know if it did, but it’s fixed!